docs/rototiller_task_reference.md

```diff
@@ -23,6 +23,9 @@
 <a name="rototiller_task:add_env"></a>
 ### #add_env
 * adds an arbitrary environment variable for use in the task
+* If the parent does call the `name=` method and the method `default=` is not called under `add_env` the value passed to `name=` is the default
```

Comment:
i think we need a short description near here of what "parent" means in this context.  
'object on which we're calling the add_env method' and not say a parent task